### ISSUE TITLE
Add Cypress.spec to the driver

### ISSUE THREAD

**Issue by brian-mann** [cypress-io/cypress#1918]
We should yield you the spec because it may be useful in the driver.

```bash
cypress run --spec cypress/integration/foo.spec.js
```

```js
const spec = Cypress.spec

// return
{
  name: 'foo.spec.js',
  relative: 'cypress/integration/foo.spec.js',
  absolute: '/Users/bmann/Dev/my-app/cypress/integration/foo.spec.js'
}
```

**Comment by brian-mann** [comment]
@bahmutov need your help adding typescript typings to this

**Comment by bahmutov** [comment]
Got it

**Comment by bahmutov** [comment]
Added

**Comment by brian-mann** [comment]
Fixed by https://github.com/cypress-io/cypress/pull/1858

**Comment by brian-mann** [comment]
Released in [`3.0.2`](https://docs.cypress.io/guides/references/changelog.html#3-0-2).
